db/migrate/20171205103643_add_primary_key_to_team_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddPrimaryKeyToTeamUser < ActiveRecord::Migration[5.1]
+  def change
+    add_column :team_users, :id, :primary_key
```

Comment:
Why do you need this column?

Reply:
that's an internal requirement of `ActiveRecord` used to perform updates on the model

Reply:
Okay... and you probably need to update in terms of `is_owner` property?